### PR TITLE
Adding in a 'how to fix a quirk' with the old bootstrapper

### PR DESCRIPTION
**Pull request by crutkas**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/powertoys/install.md [hunk 1]
```diff
@@ -75,6 +75,12 @@ cd $Env:WIX\"bin"
 .\dark.exe -x ${Env:\USERPROFILE}"\Desktop\extractedPath" ${Env:\USERPROFILE}"\Desktop\PowerToysSetup-0.53.0-x64.exe"
 ````
 
+### Fixes for uninstalling 0.51 and earlier builds issues
+
+If you have hit an issue where the MSI could have been cleaned up, you can grab the installer via the [PowerToys release page](https://github.com/microsoft/PowerToys/releases) and then run the command.  You'll want to change the EXECUTABLE_INSTALLER_NAME to what the file name actually is.
+
+In PowerShell, run `.\EXECUTABLE_INSTALLER_NAME.exe --extract_msi` and this will extract the MSI to your desktop.
```
Comment (Jay-o-Way) [inline hub/powertoys/install.md:82]: Path does not need to be specified at all?

File: hub/powertoys/install.md [hunk 2]
```diff
@@ -75,6 +75,12 @@ cd $Env:WIX\"bin"
 .\dark.exe -x ${Env:\USERPROFILE}"\Desktop\extractedPath" ${Env:\USERPROFILE}"\Desktop\PowerToysSetup-0.53.0-x64.exe"
 ````
 
+### Fixes for uninstalling 0.51 and earlier builds issues
+
+If you have hit an issue where the MSI could have been cleaned up, you can grab the installer via the [PowerToys release page](https://github.com/microsoft/PowerToys/releases) and then run the command.  You'll want to change the EXECUTABLE_INSTALLER_NAME to what the file name actually is.
```
Comment (Jay-o-Way) [inline hub/powertoys/install.md]:
```suggestion
If you have an issue where the MSI is not accessible, you can download the installer, that corresponds with the installed version, via the [PowerToys release page](https://github.com/microsoft/PowerToys/releases) and then run the following command. You'll want to change the EXECUTABLE_INSTALLER_NAME to what the file name actually is.
```

File: hub/powertoys/install.md (hunk 1 above)
Comment (crutkas) [reply hub/powertoys/install.md:82]: At a certain point, we can only help so much.

**Comment by crutkas** [comment]
@mattwojo this should be good to go

**Comment by PRMerger18** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**